src/main/java/com/scaleset/cfbuilder/ec2/Instance.java

```diff
@@ -28,4 +28,7 @@ default Instance name(String name) {
 
     Instance userData(Object userData);
 
+    // CFNInit
```

Comment:
Is this comment really necessary?

Reply:
Yes but i will rephrase it. The first part only adds pieces to the property part while the `addCFNInit ` creates a whole new part on the same level as properties